### ISSUE TITLE
element-react webpack bundle reduce size tips & hints

### ISSUE THREAD

**Issue by juanpicado** [eleme/element-react#490]
### Description

I'm using your library with`webpack` but despite the low number of components I'm using, `element-react` into the bundle is 1mb. 

![screen shot 2017-07-15 at 10 56 51 pm](https://user-images.githubusercontent.com/558752/28242671-2c44718c-69b2-11e7-8530-f17e4c7379c8.png)

I had

```
import {Loading, MessageBox} from 'element-react';
```
which I replaced by

```
import Loading from 'element-react/Loading';
import MessageBox from 'element-react/MessageBox';
```

but `webpack` is not able to locate them. I looked into node_modules/element-react and the structure is pretty hard to follow and not intuitive in order to include a simple component.

`lodash` for instance allow this, allowing me to save size in my final bundle.

```
import isString from 'lodash/isString';
```

### Reproduce Steps

1. Any webpack dummy bundle using element-react

### Error Trace (if possible)

```
ERROR in ./src/webui/src/modules/home/index.js
Module not found: Error: Can't resolve 'element-react/MessageBox' in '/src/webui/src/modules/home'
 @ ./src/webui/src/modules/home/index.js 50:18-53
 @ ./src/webui/src/app.js
 @ ./src/webui/src/index.js

ERROR in ./src/webui/src/modules/home/index.js
Module not found: Error: Can't resolve 'element-react/Loading' in '/src/webui/src/modules/home'
 @ ./src/webui/src/modules/home/index.js 46:15-47
 @ ./src/webui/src/app.js
 @ ./src/webui/src/index.js
````

### Solution

A ES6 friendly way to include selectively components without include the whole package.

**Comment by e1emeb0t** [comment]
Appreciate for the details, can you try webspack's [treeshaking](https://webpack.js.org/guides/tree-shaking/) feature since we had published the ES6 code into the npm package, you can see `jsnext:main` in the package.json.

**Comment by e1emeb0t** [comment]
Webpack without tree shaking
======================
![untitled](https://user-images.githubusercontent.com/21698993/28256709-774460e8-6af7-11e7-9239-8a985a36bf14.png)

webpack with tree shaking
======================
![untitled2](https://user-images.githubusercontent.com/21698993/28256721-894b54d6-6af7-11e7-8750-2d669dfbb746.png)

Result
====
1.2Mb bundle reduced into 939kb, about 75%.

**Comment by juanpicado** [comment]
I'll give it a try and I'll let you know

**Comment by e1emeb0t** [comment]
I just updated the comment, you can follow the link I pointed above.

**Comment by e1emeb0t** [comment]
Tried with Webpack Visualizer

<img width="729" alt="untitled" src="https://user-images.githubusercontent.com/21698993/28256983-deaf9854-6af9-11e7-9a00-a6b5ad57cbbc.png">

**Comment by e1emeb0t** [comment]
Another result from `source-map-explorer`

<img width="1279" alt="wx20170717-174814 2x" src="https://user-images.githubusercontent.com/21698993/28263074-12d051fa-6b18-11e7-8dfb-d5869d7a3c7e.png">

<img width="1279" alt="wx20170717-174845 2x" src="https://user-images.githubusercontent.com/21698993/28263095-244abf42-6b18-11e7-931b-725b99ba6789.png">
